frontend-api/src/main/java/uk/gov/di/authentication/frontendapi/lambda/LoginHandler.java

```diff
@@ -195,6 +194,27 @@ public APIGatewayProxyResponseEvent handleRequestWithUserContext(
                                 codeStorageService.getIncorrectPasswordCount(request.getEmail())),
                         pair("attemptNoFailedAt", configurationService.getMaxPasswordRetries()));
 
+                if (incorrectPasswordCount + 1 >= configurationService.getMaxPasswordRetries()) {
```

Comment:
looking at this again, think we need to move it into the invalid credentials block, as that's the first time you know when the account is locked